packages/react-search-ui-views/src/styles/themes/reference-ui/components/_select.scss

```diff
@@ -4,7 +4,7 @@
   font-size: 0.875rem;
   margin-top: $sizeS;
   width: 100%;
-  min-width: 60px;
+  min-width: 65px;
```

Comment:
I'm concerned that since we hardcode the width here, we could eventually face an issue where a font with even wider letters would cause ellipsis to appear. Is it possible to make a content define the width of select?

Reply:
I'm poking around at this now and it looks like it's `react-select`'s default styles/css causing this behavior, which is a bit disappointing. For whatever reason they're absolutely positioning the select text (even when they already have `display: flex` and `align-items: center`going on) and also manually setting text-ellipses, overflow-hidden, etc.

I think my preference would be to override/unset react-select's styles, and then tweak their padding/margins, like so:

<img width="1259" alt="" src="https://user-images.githubusercontent.com/549407/60530602-f4b09800-9cad-11e9-921b-58b76263bca3.png">

Which would then allow the select box to grow normally to accommodate all widths & fonts without truncating.

<img width="148" alt="" src="https://user-images.githubusercontent.com/549407/60530622-03974a80-9cae-11e9-99d2-20e5195bc04b.png">

Reply:
OK, I think I something working locally for this.

https://github.com/elastic/search-ui/blob/4d555722b6c3fe15f15fb6bc8aee54ac7b893297/packages/react-search-ui-views/src/ResultsPerPage.js#L5-L10

can be expanded to include:

```js
const setDefaultStyle = {
  option: () => ({}),
  control: () => ({}),
  dropdownIndicator: () => ({}),
  indicatorSeparator: () => ({}),
  singleValue: provided => {
    // Pulling out CSS that we don't want
    const { position, top, transform, maxWidth, ...rest } = provided;
    return { ...rest, lineHeight: 1, marginRight: 0 };
  },
  valueContainer: provided => ({ ...provided, paddingRight: 0 })
};
```

Which allows us to get rid of the `min-width` CSS above and generates a more responsive select, like so.

<img width="148" alt="" src="https://user-images.githubusercontent.com/549407/60531813-a94bb900-9cb0-11e9-95f0-ae4d30360b33.png">

Reply:
Oh wow, thank you @constancecchen. Providing a fix saved me a huge headache.

I put the change in place, mind taking a look?

Reply:
I can confirm this looks great to me both locally and on storybook! Will approve here once tests are passing!